services/tenant-ui/frontend/src/components/innkeeper/reservation/RefreshPassword.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <Button
+    :label="$t('reservations.refreshPassword')"
+    class="p-button-rounded p-button-text"
+    icon="pi pi-refresh"
+    @click="refresh"
+  />
+</template>
+
+<script setup lang="ts">
+// Vue
```

Comment:
I feel like the import categories are getting a bit out of control. I know it's like this in other places. I started doing my imports in 2 categories. Libraries and source code. You don't have to change this I just want to mention it.